docs/troubleshoot/troubleshoot-apiml.md

```diff
@@ -232,6 +232,31 @@ This is quite a complex area and you should ask your systems programmer for advi
 
 class profile SDSF ISF.CONNECT.\*\* (G)
 
+## Error messages from TCP/IP
+
+On starting Zowe, you may see the following messages in `DDNAME SYSPRINT`:
+```
+2025-09-02 11:05:10 user ERROR (gateway-service,bin/validate.sh:14) - Netstat test fail with exit code 16 (EZZ2376I Could not determine TCPIPjobname, using default of 'INET'
+2025-09-02 11:05:10 user ERROR (gateway-service,bin/validate.sh:14) - EZZ2377I Could not establish affinity with INET (1011/11B3005A) - can not provide the requested option information)
+```
+
+or the following message:
+
+```
+2025-09-02 11:05:10 user ERROR (gateway-service,bin/validate.sh:14) Netstat test fail with exit code 255 (EZZ2385I Access to Netstat -c denied - SAF RC is 00000008)
```

Comment:
Is there any explanation for when a user receives this message? You mention the first case...